8Knot/pages/contributors/visualizations/active_drifting_contributors.py

```diff
@@ -292,9 +286,7 @@ def create_figure(df_status: pd.DataFrame, interval):
         )
 
         # edit hover values
-        fig.update_traces(
-            hovertemplate=hover + "<br>Contributors: %{y}<br>" + "<extra></extra>"
-        )
+        fig.update_traces(hovertemplate=hover + "<br>Contributors: %{y}<br>" + "<extra></extra>")
```

Comment:
ugh i hate this change but i know its the linter

Reply:
yeah I agree